supabase/migrations/11_stats.sql

```diff
@@ -63,13 +63,15 @@ comment on column catalog_stats.flow_document is
 do $$
 begin
     if not exists (select from pg_catalog.pg_roles where rolname = 'stats_loader') then
-        create role stats_loader with login password 'stats_loader_password';
+        create role stats_loader with login password 'stats_loader_password' bypassrls;
    end if;
 end
 $$;
 
+-- stats_loader loads directly to the catalog_stats table. Postgres routes records to the correct
```

Comment:
Can we leave the owner as postgres? It seems strange that everything _but_ this is owned by postgres. The bypassrls attribute should allow stats_loader to load into it without invoking the RLS policy.

We'd still need to grant insert / select / update on catalog_stats to stats_loader.

Reply:
I had to change the owner to be `stats_loader`, otherwise the materialization(s) will fail when they try to add comments to the table if the user is not the owner of the able. I also think it is strange for postgres not to own this table though so I'll see if I can figure something else out.

Reply:
The handling of the target table is in the [connector code](https://github.com/estuary/connectors/blob/24c0a6ef9c0dd523121e66250c00b0362ad43f15/materialize-postgres-rc/sqlgen.go#L71-L92). Adding table & column comments happens no matter what, even if the target table already exists. Ideally I think the connector would leave the table & column comments alone if the target table already exists, but I don't know if this can be easily done. It also doesn't look like there is any way to get around the user being the owner of the table for adding comments, see https://www.postgresql.org/docs/current/sql-comment.html

I could imagine the connector providing an option to user a different templated query for not adding comments. That doesn't seem great either, but maybe the best we can do. What do you think @jgraettinger ?

Reply:
Yep I agree with your conclusion. I think the answer for my opening question is "no". Mind updating the migration with a comment summarizing why we're deviating from the norm here? Then LGTM